### ISSUE TITLE
Legal update option issue

### ISSUE THREAD

**Issue by ghost** [humhub-contrib/legal#12]
When I want to update my terms of use, I use the legal update option. However, this only works once. When I want to make an update again, the legal update does not require approval from users, it does not work.

**Comment by luke-** [comment]
@yurabakhtin Can you please check on occasion why the reset button does not work? It is important that the "Legal Update" tab is filled with text, but it still doesn't work for me.

**Comment by yurabakhtin** [comment]
@luke- As I understood you tell about the red button "Reset confirmation" on the page `/legal/admin/page?pageKey=terms`.
I see the button removes records only from DB table `contentcontainer_setting` for all users, i.e. it makes to again accept new terms for all users.
But when a user clicks "Accept" this code also memorizes this action in Sessions - https://github.com/humhub-contrib/legal/blob/master/Events.php#L107. And for checking firstly the flag is used here https://github.com/humhub-contrib/legal/blob/master/Events.php#L76. It means the user will be requested to accept new terms only after next log in action when Sessions will be cleared. I tested and it works as I described.

If we want to request new/reset terms immediate even if a user already accepted previous/old terms then we should remove the flag from Sessions and keep only in DB, but I think the current behaviour is normal, and it seems the cache in Sessions was implemented for optimization.
